src/Core/Report/GAEFlexMetadataProvider.php

```diff
@@ -86,4 +93,18 @@ public function versionId()
     {
         return $this->data['versionId'];
     }
+
+    /**
+     * Return the labels. We need to evaluate $_SERVER for each request.
+     * @return array
+     */
+    public function labels()
+    {
+        if (isset($_SERVER['HTTP_X_CLOUD_TRACE_CONTEXT'])) {
```

Comment:
Should we do this in __construct where we're passed in the $server array and then just return it here?